.github/workflows/exception-codes.yml

```
@@ -0,0 +1,74 @@
+name: exception-codes
+
+on:
+  schedule:
+    - cron:  '15 */1 * * *'
+  workflow_dispatch:
+
+jobs:
+  exception-code-pages:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # online, offline
+        mode: [ 'online' ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

```suggestion
        uses: actions/checkout@v4
```

Use the latest version of checkout.

---

done